### PR TITLE
Add override params

### PR DESCRIPTION
**Pull request by ronakfof**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

**Review by angelonakos — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +22,16 @@ The only required input is `project-name`.
...
    that you configured in the CodeBuild project.
+1. **compute-type-override** (optional) :
+   The name of a compute type for this build that overrides the one specified
+   in the build project.
+1. **environment-type-override** (optional) :
+   A container type for this build that overrides the one specified in the
+   build project.
+1. **image-override** (optional) :
+   The name of an image for this build that overrides the one specified
+   in the build project. By default, the action uses the default environment
```
Comment (linghaoz) [inline README.md:33]: nit pick. Is the explanation here meant to match the public documentation? If so, I don't see the second sentence in our docs.

File: test/code-build-test.js [hunk 2]
```diff
@@ -173,6 +179,9 @@ describe("inputs2Parameters", () => {
     expect(test)
       .to.haveOwnProperty("buildspecOverride")
       .and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("computeTypeOverride").and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("environmentTypeOverride").and.to.equal(undefined);
+    expect(test).to.haveOwnProperty("imageOverride").and.to.equal(undefined);
```
Comment (linghaoz) [inline test/code-build-test.js:184]: Can we have a test case where the overrides are specified?

Comment (ronakfof) [reply test/code-build-test.js]: Resolved

File: README.md (hunk 1 above)
Comment (ronakfof) [reply README.md]: Resolved

**Review by linghaoz — DISMISSED** [review]

**Review by subinataws — APPROVED** [review]

File: test/code-build-test.js [hunk 3]
```diff
@@ -173,6 +179,67 @@ describe("inputs2Parameters", () => {
...
+      .and.to.equal(undefined);
+    expect(test)
+      .to.haveOwnProperty("computeTypeOverride")
+      .and.to.equal(`BUILD_GENERAL1_LARGE`);
+    expect(test)
+      .to.haveOwnProperty("environmentTypeOverride")
+      .and.to.equal(`LINUX_CONTAINER`);
+    expect(test)
+      .to.haveOwnProperty("imageOverride")
+      .and.to.equal(`111122223333.dkr.ecr.us-west-2.amazonaws.com/codebuild-docker-repo`);
```
Comment (linghaoz) [inline test/code-build-test.js:242]: this doesn't match the input on line 221

Comment (ronakfof) [reply test/code-build-test.js:242]: Resolved

**Review by linghaoz — APPROVED** [review]

**Pull request merged**